npm/cmd/start.go

```diff
@@ -119,6 +121,12 @@ func start(config npmconfig.Config, flags npmconfig.Flags) error {
 
 	k8sServerVersion := k8sServerVersion(clientset)
 
+	nodeName := models.GetNodeName()
+	err = labelNode(clientset, nodeName, util.InstalledLabelValue)
+	if err != nil {
+		metrics.SendErrorLogAndMetric(util.NpmID, "error: failed to label node as NPM installed. err: %s", err.Error())
```

Comment:
should this exit on error? or maybe retry with exponential backoff + jitter? worried that we won't label the node if apiserver is temporarily down.

Reply:
good catch. Let's exit on error